global/prometheus-alertmanager/slack.tmpl

```diff
@@ -25,6 +25,7 @@
 {{ if .CommonLabels.sentry }}*<https://sentry.{{ .CommonLabels.region }}.cloud.sap/monsoon/{{ .CommonLabels.sentry }}|Sentry>* {{ end -}}
 {{ if .CommonLabels.playbook }}*<https://operations.global.cloud.sap/{{ .CommonLabels.playbook }}|Playbook>* {{ end -}}
 {{ if .CommonLabels.kibana }}*<https://logs.{{ .CommonLabels.region }}.cloud.sap/{{ .CommonLabels.kibana }}|Kibana>* {{ end -}}
+{{ if .CommonLabels.cloudops }}*https://dashboard.{{ .CommonLabels.region }}.cloud.sap/ccadmin/cloud_admin/cloudops#/universal-search/{{ .CommonLabels.cloudops }}|CloudOps>* {{ end -}}
```

Comment:
I think this is the wrong location. The link needs to be per alert similar as the link to prometheus. The `cloudops` can ( and will ) contain different content for each alert.

Reply:
true. moved cloudops, sentry link up